src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private File fileRead;
+    private File fileWrite;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        fileRead = new File(fromFileName);
+        fileWrite = new File(toFileName);
+        getRead(fileRead);
+        getWrite(fileWrite);
+    }
+
+    public String getRead(File file) {
```

Comment:
I also suggest splitting this method into two methods, one that reads data from the file and one that receives this data and creates a report based on it